src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -2,40 +2,78 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Collections.Generic;
+using System.IO;
 using System.Linq;
+using System.Net;
 using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Docs.Build
 {
     internal static class OpsConfigAdapter
     {
-        private static readonly bool s_prod = !string.Equals(
+        private const string MonikerDefinitionApi = "https://ops/monikerDefinition/";
+        private const string MetadataSchemaApi = "https://ops/metadataschema/";
+        private const string MarkdownValidationRulesApi = "https://ops/markdownvalidationrules/";
+
+        private static readonly string s_opsToken = Environment.GetEnvironmentVariable("DOCS_OPS_TOKEN");
+        private static readonly IReadOnlyDictionary<string, string> s_opsHeaders = new Dictionary<string, string>
+        {
+            { "X-OP-BuildUserToken", s_opsToken },
+        };
+
+        private static readonly bool s_isProduction = !string.Equals(
             "PPE", Environment.GetEnvironmentVariable("DOCS_ENVIRONMENT"), StringComparison.OrdinalIgnoreCase);
```

Comment:
suggestion: better change to equals("prod") for checking is_production, considering passing 'sandbox' by mistake.

Reply:
The intention is to be `prod` by default, the logic is correct but not intuitive, maybe `is_sandbox` expressed it better.